adjust/kpis_test.go

```diff
@@ -2,45 +2,20 @@ package adjust
 
 import (
 	"context"
+	"fmt"
 	"net/http"
 	"reflect"
 	"testing"
 )
 
-func TestKPIService_List(t *testing.T) {
-	client, mux, _, teardown := setup()
-	t.Log("Setup Done")
-	defer teardown()
-
-	mux.HandleFunc("/kpis/v1/random.json", func(w http.ResponseWriter, r *http.Request) {
-		testMethod(t, r, "GET")
-		w.WriteHeader(http.StatusOK)
-		w.Write(loadFixture("kpis.json"))
-	})
-	opt := Options{}
-	got, _, err := client.KPI.List(context.Background(), &opt)
-	if err != nil {
-		t.Errorf("KPI.List returned error: %v", err)
-	}
-
-	want := &KPI{}
-	responseToInterface(loadFixture("kpis.json"), &want)
-	if !reflect.DeepEqual(got, want) {
-		t.Errorf("KPI.List = %+v, want %+v", got, want)
-	}
-}
-
-func TestKPIService_ListWithOptions(t *testing.T) {
-	client, mux, _, teardown := setup()
-	t.Log("Setup Done")
-	defer teardown()
-
-	mux.HandleFunc("/kpis/v1/random.json", func(w http.ResponseWriter, r *http.Request) {
-		testMethod(t, r, "GET")
-		w.WriteHeader(http.StatusOK)
-		w.Write(loadFixture("kpis_options.json"))
-	})
-	opt := Options{
+var kpiTest = []struct {
```

Comment:
Would call this var testCases (or maybe kpiTestCases) to underline that there are several different cases, similar to here: https://blog.golang.org/subtests